modules/kubientBidAdapter.js

```diff
@@ -0,0 +1,115 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'kubient';
+const END_POINT = 'https://kssp.kbntx.ch/pbjs';
+const VERSION = '1.0';
+const VENDOR_ID = 794;
+export const spec = {
+  code: BIDDER_CODE,
+  gvlid: VENDOR_ID,
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid: function (bid) {
+    return !!(bid && bid.params);
+  },
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (!validBidRequests || !bidderRequest) {
+      return;
+    }
+    var result = validBidRequests.map(function (bid) {
+      let data = {
+        v: VERSION,
+        requestId: bid.bidderRequestId,
+        adSlots: [{
+          bidId: bid.bidId,
+          zoneId: bid.params.zoneid || '',
+          floor: bid.params.floor || 0.0,
+          sizes: bid.sizes || [],
+          schain: bid.schain || {},
+          mediaTypes: bid.mediaTypes
+        }],
+        referer: (bidderRequest.refererInfo && bidderRequest.refererInfo.referer) ? bidderRequest.refererInfo.referer : null,
+        tmax: bidderRequest.timeout,
+        gdpr: (bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies) ? 1 : 0,
+        consent: (bidderRequest.gdprConsent && bidderRequest.gdprConsent.consentString) ? bidderRequest.gdprConsent.consentString : null,
+        consentGiven: kubientGetConsentGiven(bidderRequest.gdprConsent),
```

Comment:
Check that `bidderRequest.gdprConsent` is defined or set default param value in `kubientGetConsentGiven`